### PR TITLE
[fr]: Updated translations

### PR DESCRIPTION
**Pull request by wblondel**
Updated as many translations as I can. It's a bit difficult to get the context without access to the paid plugins. I haven't updated the translations for which I'm not sure of the context.

### CODE REVIEW

File: locales/fr/php.json [hunk 1]
```diff
@@ -130,44 +130,44 @@
...
-    "team_slug": "Team Slug",
-    "team_trial": "Team Trial",
-    "team_trial_will_expire_on": "The team's trial period will expire on :date.",
-    "teams": " Teams",
-    "teams_currently_trialing": "Teams Currently Trialing",
+    "team": " Equipe",
+    "team_billing": "Facturation d'équipe",
+    "team_members": "Membres de l'équipe",
+    "team_name": " Nom de l'équipe",
+    "team_name_was_updated": " Le nom de votre équipe a été modifié!",
```
Comment (caouecs) [inline locales/fr/php.json:141]: a space between "modifié" and "!"

File: locales/fr/php.json [hunk 2]
```diff
@@ -130,44 +130,44 @@
...
     "url": "Le format de l'URL de :attribute n'est pas valide.",
     "user": "Aucun utilisateur n'a été trouvé avec cette adresse email.",
-    "user_already_invited_to_team": "That user is already invited to the team.",
-    "user_already_on_team": "That user is already on the team.",
-    "user_doesnt_belong_to_team": "The user does not belong to the given team.",
-    "user_invited_to_join_team": ":UserName has invited you to join their team!",
+    "user_already_invited_to_team": "Cet utilisateur est déjà invité à rejoindre l'équipe.",
+    "user_already_on_team": "Cet utilisateur est déjà dans l'équipe.",
+    "user_doesnt_belong_to_team": "L'utilisateur n'appartient pas à l'équipe donnée.",
+    "user_invited_to_join_team": ":UserName vous a invité à joindre son équipe!",
```
Comment (caouecs) [inline locales/fr/php.json:164]: a space between "équipe" and "!"

**Review by caouecs — CHANGES_REQUESTED** [review]
Merci

File: locales/fr/php.json (hunk 2 above)
Comment (wblondel) [reply locales/fr/php.json]: Fixed, thanks!

File: locales/fr/php.json (hunk 1 above)
Comment (wblondel) [reply locales/fr/php.json]: Fixed, thanks!

**Comment by wblondel** [comment]
Hi, sorry I didn't mean to remove request for @WhereIsLucas , I don't know how that happened. I pushed the requested changes.

**Comment by andrey-helldar** [comment]
@wblondel, don't worry, I fixed it 🤝

**Comment by andrey-helldar** [comment]
@caouecs, @WhereIsLucas, ping

**Comment by andrey-helldar** [comment]
@caouecs, @WhereIsLucas, hi! Can you check this PR?

File: locales/fr/php.json [hunk 3]
```diff
@@ -130,44 +130,44 @@
...
-    "we_found_invitation_to_team": "We found your invitation to the :teamName team!",
-    "wheres_your_team": "Where's Your Team?",
-    "you_have_been_invited_to_join_team": "You have been invited to join the :teamName team!",
-    "you_have_x_invitations_remaining": " You currently have :count invitation(s) remaining.",
-    "you_have_x_teams_remaining": "You currently have :teamCount teams remaining."
+    "vat_id": "Ce numéro de TVA n'est pas valide.",
+    "view_all_teams": "Voir toutes les équipes",
+    "we_found_invitation_to_team": "Nous avons trouvé votre invitation à rejoindre l'équipe :teamName !",
+    "wheres_your_team": "Où est votre équipe ?",
+    "you_have_been_invited_to_join_team": "Vous avez été invité à rejoindre l'équipe :teamName !",
```
Comment (caouecs) [inline locales/fr/php.json:170]: Ouf j'ai trouvé un truc à dire ;)

Vous avez été invité(e) à rejoindre l'équipe :teamName !

**Review by caouecs — CHANGES_REQUESTED** [review]
Juste une modification pour inclure tout le monde et ça ira pour moi

File: locales/fr/php.json (hunk 3 above)
Comment (wblondel) [reply locales/fr/php.json]: C'est fait, j'ai corrigé. Egalement dans le fichier json.json

**Comment by wblondel** [comment]
I applied the requested change, and I also translated additional strings

**Comment by caouecs** [comment]
merci

**Pull request merged**